### ISSUE TITLE
Evaluation results

### ISSUE THREAD

**Issue by Andreas71** [tensorflow/models#4019]
Hello,

I trained my model using my own training dataset but when i try to run the evaluation it only displays the test images with the detections in the tensorboard, but nothing about the MaP. How can i display the MaP or any other accuracy result?

Thanks in advance

**Comment by k-w-w** [comment]
What command did you use to run the model?

**Comment by Andreas71** [comment]
First,
python eval.py \
    --logtostderr \
    --pipeline_config_path=training/ssd_mobilenet_v1_pets.config \
    --checkpoint_dir=training/ \
    --eval_dir=eval/

And then to visualise it
tensorboard --logdir='eval'

In tensorboard only the images with the detections appear and nothing about the MaP

**Comment by k-w-w** [comment]
derekjchow can you help with this?

**Comment by Andreas71** [comment]
Here is a screenshot of the tensorboard. It doesnt show anything about MaP

![sce](https://user-images.githubusercontent.com/24529098/39022680-924fce02-442e-11e8-91ab-e5da346fa162.png)

**Comment by nguyenkhaithinh** [comment]
I have the same issue as well. When I inspect the eval event.out files (tensorboard --inspect --logdir=./), it shows:

======================================================================

Found event files in:
./

These tags are in ./:
audio -
histograms -
images
   image-1
   image-3
   image-5
   image-7
   image-9
scalars -
tensor -
======================================================================

Event statistics for ./:
audio -
graph -
histograms -
images
   first_step           18448
   last_step            18448
   max_step             18448
   min_step             18448
   num_steps            1
   outoforder_steps     []
scalars -
sessionlog:checkpoint -
sessionlog:start -
sessionlog:stop -
tensor -
======================================================================

I am using tensorflow-gpu (1.7.0) and tensorflow-tensorboard (0.4.0rc3).

**Comment by Andreas71** [comment]
@k-w-w  @derekjchow any help?

**Comment by derekjchow** [comment]
You need to run the [evaluation job](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/running_locally.md#running-the-evaluation-job) to see metrics such as mAP.

**Comment by Andreas71** [comment]
i used that as stated in a previous answer but it doesnt show me any metrics in the tensorboard

python eval.py 
--logtostderr 
--pipeline_config_path=training/ssd_mobilenet_v1_pets.config 
--checkpoint_dir=training/ 
--eval_dir=eval/

**Comment by nowgood** [comment]
add evaluation metrics to `eval_config` in the pipline.config file, such as

```
eval_config: {
  num_examples:2000
  max_evals: 10
  eval_interval_secs: 5
  metrics_set: "coco_detection_metrics"
}

```

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
